example/simple_testing/c/Wrapper.h

```diff
@@ -0,0 +1,170 @@
+#ifndef Wrapper_H
+#define Wrapper_H
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stddef.h>
+#include <stdbool.h>
+#include "diplomat_runtime.h"
+
+#include "TestingStruct.d.h"
+
+#include "Wrapper.d.h"
+
+
+
+
+
+
+typedef struct DiplomatCallback_Wrapper_test_multi_arg_callback_f {
+    const void* data;
+    int32_t (*run_callback)(int32_t(*cb_pointer)(int32_t), int32_t );
+    void (*destructor)(const void*);
+} DiplomatCallback_Wrapper_test_multi_arg_callback_f;
+
+int32_t run_create_DiplomatCallback_Wrapper_test_multi_arg_callback_f_callback(int32_t(*cb_pointer)(int32_t), int32_t arg0 ) {
+    return cb_pointer(arg0);
+}
+
+DiplomatCallback_Wrapper_test_multi_arg_callback_f C_create_DiplomatCallback_Wrapper_test_multi_arg_callback_f(int32_t(*cb_pointer)(int32_t)) {
```

Comment:
question: so this is just a convenience function for C users, yes?

Reply:
yes -- removed now